### PR TITLE
Add a templatetag which escapes only the characters Slack wants to escape.

### LINKED ISSUE #33: Over-aggressive HTML entity escaping
By default, `django-slack` escapes all HTML entites (well really Django templates do, but that's essentially the same thing here). This is [documented](https://django-slack.readthedocs.org/#usage), but the documentation should probably be expanded to say that [only `<`, `>` and `&` should be escaped](https://api.slack.com/docs/formatting#how_to_escape_characters). In fact the slack documentation says NOT to escape other characters:

> That's it. **Don't HTML entity-encode the entire message.** Of course, when you send your message using URL-encoding, you'll still need to URL encode the ampersand.

Specifically, I was running into an issue where I had a message with a single quote (`'`) that was showing up as an entity in slack (`&#39;`). The examples in the documentation show usage of the [`safe` templatetag](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#safe), but it was not clear to me why this was happening. Calling this out explicitly would be nice.

I'm unsure if this is a documentation or a code issue, but the work around is to pass variables through the `safe` templatetag or disable `autoescape`.

### PR DESCRIPTION
**Pull request by clokep**
* Add a `escapeslack` template tag based on the `escapejs` tag.
* Adds a test application (run `python manage.py test`)

I was unable to figure out how to automatically inject this into each template rendered by `django-slack`. Any ideas of where to look?

Fixes #33

### CODE REVIEW

**Comment by lamby** [comment]
> I was unable to figure out how to automatically inject this into each template rendered by django-slack. Any ideas of where to look?

*Completely* untested but I was thinking something replacing ``render_to_string`` with

```python
from django.template import Engine
engine = Engine(builtins=['django_slack.templatetags.django_slack'])
engine.render_to_string(...)
```

**Comment by clokep** [comment]
Interesting idea! I'll take a look at that this afternoon.

**Comment by clokep** [comment]
@lamby I pushed another commit which uses an `Engine` based on your suggestion. Let me know what you think.

File: django_slack/api.py [hunk 1]
```diff
@@ -2,11 +2,14 @@
 
 from django.conf import settings
 from django.template.loader import render_to_string
```
Comment (lamby) [inline django_slack/api.py]: Can drop this import now, no?

Comment (clokep) [inline django_slack/api.py]: Good catch! Done.

**Comment by lamby** [comment]
Thanks so much :)  Merged with some cosmetic changes

**Comment by clokep** [comment]
Great, thanks for the quick feedback! :+1:

**Comment by clokep** [comment]
Uh-oh...this seems to break finding templates outside of the `django_slack` package...

**Comment by lamby** [comment]
Damn. Yeah, that makes sense. We *could* hack this to inject it temporarily into the builtins.. but lets not.

**Comment by lamby** [comment]
Fixed and released as ``5.0.0``

**Comment by clokep** [comment]
I agree, just using the `{% load escapeslack %}` and not being too clever about it is the way to go. Thanks for fixing it so fast! (Fixed in d103fcf2b9896ce755758f0ac5bb30cde1a2d4e5 if anyone sees this and is curious.)

**Pull request merged**